iroh-sync/src/store.rs

```diff
@@ -132,6 +140,24 @@ pub trait Store: std::fmt::Debug + Clone + Send + Sync + 'static {
     /// Get all content hashes of all replicas in the store.
     fn content_hashes(&self) -> Result<Self::ContentHashesIter<'_>>;
 
+    /// Get the latest timestamp and key for each author in a namespace
+    fn get_latest(&self, namespace: NamespaceId) -> Result<Self::LatestIter<'_>>;
```

Comment:
Isn't this the same as `get_all` without the values? Might be clearer to call this `get_all_keys()` or similar

Reply:
No, this only gets *one entry per author*, not *one entry per key*. Will try to improve the docs.

Reply:
ohh, I see, I thought this was per author & per key